tsconfig.json

```diff
@@ -10,6 +10,14 @@
 		"resolveJsonModule": true,
 		"target": "esnext",
 
+		/* Project Imports */
+		"baseUrl": ".",
+		"paths": {
+			"@wordpress/*": [
+				"packages/*/src/index.js"
```

Comment:
Not sure if this was intentional or not, but you might want to consider leaving off the `index.js` portion to allow importing from nested directories.

I think I may have recalled seeing something written somewhere that you all are purposefully not exposing stuff not explicitly exported from `index.js` though, so if that's indeed the case, then this looks good to me.

Reply:
> Not sure if this was intentional or not, but you might want to consider leaving off the `index.js` portion to allow importing from nested directories.
> 
> I think I may have recalled seeing something written somewhere that you all are purposefully not exposing stuff not explicitly exported from `index.js` though, so if that's indeed the case, then this looks good to me.

That was the original idea, yes. I'm not overly committed to the idea, so if we need to change this in the future, I'd be okay with it.